### DISCUSSION TITLE
Switch of interaction between specific atomic pair

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/deepmd-kit#1536] [Q&A]
Hi All,

I am wondering, is there some option in lammps to switch off the interaction between specific pairs of atoms with deepmd potential.

Thanks,
Mayank

**Comment by njzjz** [comment]
Yes, the option is called `exclude_types`. Details are given in the doc. The theory can be found in [my paper](https://doi.org/10.1021/acs.jctc.1c00201) (specifically, Eq. 14).

**Comment by mayankaditya** [comment]
Thanks Jinzhe Zeng. I was actually looking for an option in lammps input file, which switch-off the interaction between specific elements pair.
I think the exclude_types option is used while training the data. Does this mean the new trained potential will be equivalent to a switch of the interaction between specific pairs?

Thanks,
Mayank

**Comment by njzjz** [comment]
The potential is many-body but not the sum of pairs.

> Does this mean the new trained potential will be equivalent to a switch of the interaction between specific pairs?

No, it's still many-body, but the contribution of r_{ij} is removed, where i, j are the specific pairs of atoms.
